src/index.ts

```diff
@@ -67,6 +79,8 @@ export class JolocomSDK {
 
     // if we are running on NodeJS, then autoconfig some things if possible
     if (process && process.version) this._autoconfigForNodeJS()
+
+    LoggerRegistrar.register(this.serviceContainer, conf.logger)
```

Comment:
so this is done to be able to register different channels, however, the below seems more readable
```
container.register(`logger`, new DefaultLogger(conf.logger)
```
(signature and implementation shall be adopted)

Reply:
Please check `LoggerRegistrar::register` logic. It needs to configure logger and create separate instances for each channel and only after that - register channels in container.

Reply:
yes, it is understandable that LoggerRegistrar abstracts logger configuration based on config and its further registration on the container. No comments about the implementation, just a remark about the naming. It (Logger Registrar) is more a helper of the ServiceContainer that registers something of "type" logger. I am looking for something that in its name describes that loggers are being registered on the service container, i.e. ServiceContainerRegistrar, because we don't register anything on the logger, but on the service container.